src/components/attendee/ScheduleOverlay/ScheduleOverlay.tsx

```diff
@@ -184,30 +184,30 @@ export const ScheduleOverlay: React.FC = () => {
         );
 
         return (
-          <div key={event.id} className={CN.eventWrapper}>
-            <div className={CN.eventTitle}>
-              <span>{event.name}</span>
-              <FontAwesomeIcon
-                icon={event.isSaved ? solidBookmark : regularBookmark}
-                onClick={() => bookmarkEvent(event)}
-                className={CN.bookmarkIcon}
-              />
+          <div key={event.id} className={CN.eventContainer}>
+            <div className={CN.eventWrapper}>
+              <div className={CN.eventTitle}>
+                <span>{event.name}</span>
+                <FontAwesomeIcon
+                  icon={event.isSaved ? solidBookmark : regularBookmark}
+                  onClick={() => bookmarkEvent(event)}
+                  className={CN.bookmarkIcon}
+                />
+              </div>
+              <div>
+                <span>
+                  {formatTimeLocalised(eventStartTime({ event })) + " - "}
```

Comment:
[NOTE] I commented on the function name further down, but here I'm unsure if any of these ideas will make it more visible. Can remain as currently is though.

```tsx
                  {formatTimeLocalised(eventStartTime({ event })) + " - "}
```

or something like 

```tsx
                  {formatTimeLocalised(eventStartTime({ event }))} -
```

or even

```tsx
                  {formatTimeLocalised(eventStartTime({ event }))} 
                  {STRING_DASH}
```

Reply:
added `STRING_DASH_SPACE` [a58b99b](https://github.com/sparkletown/sparkle/pull/2905/commits/a58b99bf83fb6ed1a9b39ec673fb2882dab9e8e9)